src/App.js

```diff
@@ -4,18 +4,23 @@ import axios from 'axios';
 import HeaderComp from './Frame/HeaderComp/HeaderComp';
 import FreeWeekRotationComp from './Components/FreeWeekRotationComp/FreeWeekRotationComp';
 import SearchFieldComp from './Components/SearchFieldComp/SearchFieldComp';
+import MatchHistoryComp from './Components/MatchHistoryComp/MatchHistoryComp';
+import {getFullUrl} from './Modules/Helper';
 import "./App.css";
 
 class App extends React.Component {
 	state = {
 		champions: null,
-		username: ""
+		username: "",
+		search_error: "",
+		display_history: false,
+		account_id: ""
 	}
 
 	componentDidMount() {
 		// Get all champions from the server when app starts
 
-		let champions_url = "http://ddragon.leagueoflegends.com/cdn/6.24.1/data/en_US/champion.json"; // all champions json
+		let champions_url = "http://ddragon.leagueoflegends.com/cdn/9.14.1/data/en_US/champion.json"; // all champions json
```

Comment:
Here as well. App module shouldn't fetch isomorphic data, move it to a different module